bindings/java/src/main/com/apple/foundationdb/KeyArrayResult.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * KeyArrayResult.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
```

Comment:
We should use 2020 in all the new files here.

```suggestion
 * Copyright 2013-2020 Apple Inc. and the FoundationDB project authors
```